src/store/journal/__tests__/journal.effects.spec.ts

```diff
@@ -120,7 +120,7 @@ describe('Journal Effects', () => {
     // ASSERT
     effects.loadJournal$.subscribe((result) => {
       expect(journalProvider.getJournal).toHaveBeenCalled();
-      // TODO: Reinstate when offline functionality is done
+      // @TODO MES-7134: Reinstate when offline functionality is done
       // expect(journalProvider.saveJournalForOffline).not.toHaveBeenCalled();
```

Comment:
think this can also be enabled and TODO removed?

Reply:
enabled